setup.py

```diff
@@ -33,6 +33,7 @@
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
```

Comment:
I think we need to update line 53 as well

Reply:
updated